get_spotify_token.rb

```diff
@@ -13,7 +13,7 @@
 
 puts <<INSTRUCTIONS
 To obtain a Spotify token, please log in to Spotify in your browser and then
-* visit 'https://accounts.spotify.com/authorize?response_type=code&redirect_uri=http://localhost:3000&scope=playlist-modify-public&client_id=#{id}'
+* visit 'https://accounts.spotify.com/authorize?response_type=code&redirect_uri=http://localhost:3000&scope=playlist-modify-public playlist-read-private&client_id=#{id}'
```

Comment:
This feels like another potential data migration issue: what happens with old tokens that don't have this required scope? ...

Because this is ad hoc for now, I don't know that we have a great option; just pointing it out as the sort of change you might be careful of in the future. I'll generate the token for prod.